website/docs/cloud-docs/api-docs/no-code-provisioning.mdx

```diff
@@ -38,18 +38,23 @@ description: >-
 
 # No-Code Provisioning API
 
-The No-Code Provisioning API allows you to establish and configure a list of Terraform modules that you can use in no-code provisioning workflows within HCP Terraform. For more information on no-code modules, see [Designing No-Code Ready Modules](/terraform/cloud-docs/no-code-provisioning/module-design).
+The no-code provisioning API allows you to create, configure, and deploy Terraform modules in the no-code provisioning workflows within HCP Terraform. For more information on no-code modules, see [Designing No-Code Ready Modules](/terraform/cloud-docs/no-code-provisioning/module-design).
 
-## Allow No-Code Provisioning of a Module within an Organization
+## Allow no-code provisioning of a module
 
 `POST /organizations/:organization_name/no-code-modules`
 
-
 | Parameter            | Description                                                                                                                                                              |
 | -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `:organization_name` | The name of the organization where no-code provisioning workflows should be allowed for this module.  |
+| `:organization_name` | The name of the organization the module belongs to.  |
+
+To deploy a module using the no-code provisioning workflow, you must meet the following requirement:
```

Comment:
```suggestion
To deploy a module using the no-code provisioning workflow, the module must meet the following requirement:
```
Right? The requirements are for the module? Just to state the obvious, this really should be a separate usage doc with a "Requirements" section, but here we are.